Habitica/res/drawable/challenge_gem_add_button_disabled.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item >
+        <bitmap
+            android:src="@drawable/plus"
+            android:tint="#e1e0e3" />
```

Comment:
if you have the dev product flavor active, android studio thinks that 21 is the minSDK.